SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -27,10 +27,174 @@
 #include "Modules/ModuleManager.h"
 #include "Net/UnrealNetwork.h"
 
+#if UE_EDITOR
```

Comment:
Why is this needed?

Reply:
There is a `GEditor` usage in this file, and it's defined in `Editor.h`. Just fixing a possible build issue.

Reply:
I think we normally use WITH_EDITOR here, but I'm not actually sure why/how they're different